bin/pycbc_live

```diff
@@ -198,6 +205,157 @@ class LiveEventManager(object):
     def ingest(self, ifo, results):
         pass
 
+class MPIRotatingFileHandler(TimedRotatingFileHandler):
```

Comment:
looks like you copied this from somewhere, but please try to keep the standard style guide for pycbc, i.e. no camelCase.

Reply:
Ah, so the rollover parts came from the logging module directly? That's mostly where the inconsistencies appear to be. Is the doRollover function one of the things you pulled from the logging module?

Reply:
Yes, I cannot rename `doRollover`, it's used by some of the subclass functionality.